content/doc/book/managing/users.adoc

```diff
@@ -1,15 +1,145 @@
 ---
 layout: section
-wip: true
+title: Users
 ---
 ifdef::backend-html5[]
-:notitle:
-:description:
+:description: Learn how to manage users in Jenkins, including creating, configuring, and understanding user management.
 :author:
 :sectanchors:
 :toc:
+:toclevels: 4
 :hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
 endif::[]
 
 = Users
 
+Managing users in Jenkins involves creating, configuring, and maintaining user accounts for individuals who interact with the Jenkins instance. This page provides a step-by-step guide on how to manage users effectively.
+
+_Jenkins supports external authentication systems like GitHub, GitLab, Google, and LDAP. If an external authentication method is configured, user management may be handled outside Jenkins. This guide applies only when using Jenkins' **internal user database**._
+
+== What managing users means
+
+In Jenkins, managing users refers to the process of creating, configuring, and maintaining user accounts. This includes:
+
+- Creating new users.
+- Configuring user details such as full name, email address, and password.
+- Modifying user information.
+- Deleting users.
+
+Administrators with the appropriate permissions can perform these actions.
+
+== Accessing the users page
+
+To manage users in Jenkins, you need to have administrative privileges. Follow these steps to access the user management section:
+
+. Log in to your Jenkins instance with an account that has administrative permissions.
++
+. Navigate to the **Manage Jenkins** page from the Jenkins dashboard.
++
+image::managing/manager-users-home-page.png[Accessing Manage Jenkins page,width=800]
+
+. Select **Users** under the **Security** section.
++
+image::managing/select-users.png[Accessing the Users section,width=800]
+
+== Creating a user
+
+To create a new user in Jenkins:
+
+. On the **Users** page, select **Create User**.
++
+image::managing/create-users-click.png[Accessing the Create User button,width=800]
+
+. Fill in the required details:
+   - **Username**: A unique identifier for the user.
+   - **Password**: A secure password for the user.
+   - **Confirm Password**: Re-enter the password.
+   - **Full Name**: The full name of the user.
+   - **Email Address**: The email address of the user.
++
+image::managing/create-users.png[Accessing the Create User Page,width=800]
```

Comment:
```suggestion
image::managing/create-users.png[Accessing the Create User Page,width=700]
```
Suggesting 700 as the width because the content of the image appears much larger on the page compared to the other screenshots.